
vim.fault.DuplicateName
=======================
    :extends:

        `vim.fault.VimFault <vim/fault/VimFault.rst>`_

  A DuplicateName exception is thrown because a name already exists in the same name space.

Attributes:

    name (`str <https://docs.python.org/2/library/stdtypes.html>`_)

    object (`str <https://docs.python.org/2/library/stdtypes.html>`_)




